hadoop-hdds/interface-server/src/main/proto/ScmServerDatanodeHeartbeatProtocol.proto

```diff
@@ -368,6 +368,8 @@ message DeletedBlocksTransaction {
   repeated int64 localID = 3;
   // the retry time of sending deleting command to datanode.
   required int32 count = 4;
+  optional uint64 totalBlockSize = 5;
```

Comment:
what will be default value for old transaction in table? when its read to populate while sending to DN in getTransactions()

Reply:
For old transaction in table, it doesn't have these two new fields.  Then consumer of old transactions (mainly new DN) should verify whether the fields exists before getting the fields value.

Reply:
how its useful of differentiating before upgrade and after upgrade values? what kind of problem we are trying to handle?